src/pixelator/graph/utils.py

```diff
@@ -87,18 +87,14 @@ def components_metrics(edgelist: pd.DataFrame) -> pd.DataFrame:
         upib_count = group_df["upib"].nunique()
         tot_count = group_df["count"].sum()
         mean_count = group_df["count"].mean()
-        median_count = group_df["count"].median()
-        tot_umi = group_df["umi"].nunique()
         # Please note that we need to use observed=True
         # here upia is a categorical column, and since not
         # all values are present in all components, this is
         # required to get a correct value.
         upia_degree = group_df.groupby("upia", observed=True)["upib"].nunique()
         upia_mean_degree = upia_degree.mean()
-        upia_median_degree = upia_degree.median()
         # Same reasoning as above
         umi_degree = group_df.groupby("upia", observed=True)["umi"].count()
-        upi_umi_median = umi_degree.median()
```

Comment:
I am not sure I want median out. We don't use median that much, but it could be interesting to have.